Models/product.js

```diff
@@ -8,7 +8,8 @@ const Product = new Schema({
     price: { type: Number, default: 0 },
     category: { type: String, enum: ['Mobility', 'Hygiene', 'Apparel', 'Bedroom', 'Dining'], required: true },
     currentCondition: { type: String, enum: ['Poor', 'Fair', 'Good', 'Very Good', 'Excellent'], required: true },
-    brand: String
+    brand: String,
+    mainImage: String
```

Comment:
I am totally okay with this. Thank you Teej!

I am just hitting a problem with rendering out this string image path on the front end. I've flagged this as an issue, but I will raise a PR for that separately.